Content.Server/Chat/V2/Messages.cs

```diff
@@ -69,13 +69,13 @@ public sealed class LoocCreatedEvent(EntityUid speaker, string message) : IChatE
 public sealed class RadioCreatedEvent(
     EntityUid speaker,
     string message,
-    RadioChannelPrototype channel)
+    string channel)
```

Comment:
"Panda why are you just sending the prototype everywhere"
"I don't know, voice in my brain"

Reply:
could prob be a ProtoId too

Reply:
Maybe, just want to avoid complexity in the ingest handlers when possible, so passing through raw channel name was the easiest option